.gitignore

```diff
@@ -1,186 +1,16 @@
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-
-# User-specific files
-*.suo
-*.user
-*.sln.docstates
+###############################################################################
```

Comment:
The main `.gitignore` file has a lot more than necessary. I can keep it if desired, but I personally find the smaller subset easier to reason about.

Reply:
(It also helps expose places where things aren't ending up where expected as almost all excluded files should end up under `artifacts`).

Reply:
I agree ... I just let GitHub decide so this is better imo too.